init-project/db-user-schema-create.sql

```diff
@@ -1,93 +1,93 @@
+DROP DATABASE IF EXISTS test_schema;
 CREATE DATABASE test_schema 
     CHARACTER SET utf8mb4 
     COLLATE utf8mb4_bin;
-    
-use test_schema;
-
 
+USE test_schema;
 
 --
--- Table structure for table `business_account`
+-- Table structure for table business_account
 --
-
```

Comment:
Changes from this file are:
- Removing unnecessary quotes.
- Adding the new `status` column.